e2e/tests/adminManagesOrders_test.js

```diff
@@ -118,7 +123,10 @@ Scenario('Create EPO order', async ({ I, caseViewPage, manageOrdersEventPage })
 });
 
 Scenario('Create EPO Prevent removal order', async ({ I, caseViewPage, manageOrdersEventPage }) => {
```

Comment:
this one still uses today as the approval date time, want to change that to approvalDate as well?

Reply:
For both EPO had to keep the same dates, as this is constraint to the present and 8 days margin, so I can't use the sequence. In this way I put it in their own case, so don't need too much magic